geokot/index.py

```diff
@@ -1,1049 +0,0 @@
-#!/usr/bin/python
-# -*- coding: utf-8 -*-
```

Comment:
idea was to strip the geocoder from index.html and move to geokot/. if you can keep it until logic from both is merged it will help, there are some cool stuff like "find kilometer for highway using milestones" in this file.

Reply:
reverted back, there are also one more file were removed `route/index.py` is it ok to remove or it also planed to be used?